CMakeLists.txt

```diff
@@ -230,12 +236,16 @@ set(arbor_supported_components)
 
 # Target microarchitecture for building arbor libraries, tests and examples
 #---------------------------------------------------------------------------
-if(ARB_ARCH)
+if(ARB_ARCH STREQUAL "none")
```

Comment:
check against empty string? no need for a magic "none"

Reply:
If the `ARB_ARCH` string has been set to "", then that was a deliberate choice of the user, and the build will break because of it.